content/tutorials/how-to-withdraw-erc20/10.index.md

```diff
@@ -98,7 +98,7 @@ async function main() {
   console.log(`Running script to bridge ERC-20 to L1`);
 
   // Initialize the wallet.
-  const l1provider = new ethers.Provider(L1_RPC_ENDPOINT);
+  const l1provider = new Provider(L1_RPC_ENDPOINT);
```

Comment:
```suggestion
  const l1provider = new ethers.JsonRpcProvider(L1_RPC_ENDPOINT);
```
This should be a provider from ethers, not zksync-ethers. Please try this suggestion